src/main/java/com/pileproject/drivecommand/command/CommandBase.java

```diff
@@ -32,27 +35,28 @@ public CommandBase(CommandType type, Map<String, Object> args) {
     }
 
     /**
-     * Get its command type
+     * Get this command type.
```

Comment:
Yeah, you are right. So, how do we keep them? Do you want me to make a wiki or something?